core/src/PrognosticData.cpp

```diff
@@ -53,22 +53,32 @@ void PrognosticData::configure()
     tryConfigure(iceGrowth);
 }
 
+// Copies an HField from a source ModelArray that is either an HField or a DGField.
+void copyFiniteDifference(const ModelArray& source, ModelArray& sink)
```

Comment:
```suggestion
void copyDGMean(const ModelArray& source, ModelArray& sink)
```

You're copying the mean value of the DG field, hence the rename. Calling it finite difference is not right. I guess you were thinking about finite volumes ... but that's strictly not correct either, even if zero-order DG is almost finite volumes (but not exactly if I understand it correctly).